profiles/kentik_snmp/cisco/cisco-wlc.yml

```diff
@@ -154,6 +155,19 @@ metrics:
           coldStandbyRedundant: 6
           warmStandbyRedundant: 7
           hotStandbyRedundant: 8
+    
+  # HA Sync Status for AirOS WLCs
+  - MIB: CISCO-LWAPP-HA-MIB
+    symbols:
+      - name: cLHaAvgPeerReachLatency # This object represents average value of peer controller reachable latency in micro seconds
+        OID: 1.3.6.1.4.1.9.9.843.1.5.1
+        poll_time_sec: 60
+      - name: cLHaBulkSyncStatus #  This object represents current status of bulk sync
+        OID: 1.3.6.1.4.1.9.9.843.1.5.3
+        poll_time_sec: 60
```

Comment:
this is unnecessary as it isn't a defined golden signal and therefore won't be natively used for anomaly detection. consider removing unless you deem the metric is critical enough to be collected at this interval